symphony-bdk-http/symphony-bdk-http-webclient/src/main/java/com/symphony/bdk/http/webclient/ApiClientBuilderWebClient.java

```diff
@@ -30,47 +32,52 @@
 
 /**
  * Specific implementation of {@link ApiClientBuilder} which creates a new instance of an {@link ApiClientWebClient}.
+ *
+ * <p><b>Please note that overriding this class is an {@link org.apiguardian.api.API.Status#EXPERIMENTAL} feature that we
+ * offer to developers for {@link ApiClient} customization. The internal contract of this class (e.g. protected methods)
+ * is subject to changes in the future.</b>
  */
 @API(status = API.Status.EXPERIMENTAL)
 public class ApiClientBuilderWebClient implements ApiClientBuilder {
 
-  private final Map<String, String> defaultHeaders;
-  private String basePath;
-  private byte[] keyStoreBytes;
-  private String keyStorePassword;
-  private byte[] trustStoreBytes;
-  private String trustStorePassword;
-  private int connectionTimeout;
-  private int readTimeout;
-  private String temporaryFolderPath;
-  private String proxyHost;
-  private int proxyPort;
-  private String proxyUser;
-  private String proxyPassword;
+  private static final Logger log = LoggerFactory.getLogger(ApiClientBuilderWebClient.class);
```

Comment:
IMHO it should be called LOGGER although currently it is not the case in the other usage we have in the BDK

Reply:
Yep, we use the default logger name coming with Lombok